openpnm/core/ModelsMixin.py

```diff
@@ -45,10 +45,17 @@ def dependency_list(self):
         d = nx.algorithms.dag.lexicographical_topological_sort(dtree, sorted)
         return list(d)
 
-    def dependency_graph(self):
+    def dependency_graph(self, mode="shallow"):
```

Comment:
To be consistent with other methods, I think this arg should be ``deep=False``.  Since it's only a binary choice, accepting an open-end mode is confusing.  By using a flag, it is more obvious what it does.

Reply:
You're right, makes sense. It's fixed now.